client/extensions/woocommerce/state/sites/settings/general/handlers.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+import { areSettingsGeneralLoaded } from 'woocommerce/state/sites/settings/general/selectors';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { setError } from '../../status/wc-api/actions';
+import {
+	WOOCOMMERCE_SETTINGS_GENERAL_REQUEST,
+	WOOCOMMERCE_SETTINGS_GENERAL_REQUEST_SUCCESS,
+} from 'woocommerce/state/action-types';
+
+export const handleSettingsGeneralSuccess = ( { dispatch }, action, next, { data } ) => {
+	const { siteId } = action;
+	dispatch( {
+		type: WOOCOMMERCE_SETTINGS_GENERAL_REQUEST_SUCCESS,
```

Comment:
@DanReyLop Maybe `WOOCOMMERCE_SETTINGS_GENERAL_REQUEST_SUCCESS` isn't the best name here. Maybe `WOOCOMMERCE_SETTINGS_GENERAL_RECEIVE`?